syntax = "proto3";

message InfoMessage
{
    int64 user_id  = 1;
    string name    = 2;
    string message = 3;
}

message ChatState
{
  int32 nt_type                     = 1;
  int64 user_id                     = 2;
  repeated InfoMessage info_message = 3;
}
